src/lib/transaction.ts

```diff
@@ -188,7 +191,7 @@ export const DEFAULT_SEND_OPTIONS: Required<
  * );
  * ```
  */
-export async function sendTransactionWithRetry(
```

Comment:
Why no longer exporting this?

Reply:
I think it will be confusing people and they will think the sendTransactionWithRetry is better than sendTransaction. SendTransaction uses sendwithRetry internally